lib/src/main/java/com/diffplug/spotless/JavaVersion.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless;
+
+/**
+ * Helper class providing Java version information
+ *
+ */
+public final class JavaVersion {
```

Comment:
Let's change this to package-private.  The smaller our public API, the better :)

Reply:
Will do